tests/js/spec/views/issueList/header.spec.jsx

```diff
@@ -281,12 +293,13 @@ describe('IssueListHeader', () => {
         queryCount={13}
       />
     );
-    expect(wrapper.find('SavedSearchTab a').text()).toBe('Saved Search 13');
-    expect(wrapper.find('SavedSearchTab').prop('isActive')).toBeTruthy();
+    expect(screen.getByText('Saved Search').parentElement).toHaveTextContent(
```

Comment:
shouldn't we only check if `Saved Search 13` is visible on the page?

Reply:
The 13 is wrapped in several spans